src/drivers/spektrum_rc/spektrum_rc.cpp

```diff
@@ -50,8 +50,11 @@
 #include <uORB/topics/input_rc.h>
 
 // Snapdraogon: use J12 (next to J13, power module side)
+#if defined(__PX4_QURT)
+#define SPEKTRUM_UART_DEVICE_PATH "/dev/tty-1"
+#else
 #define SPEKTRUM_UART_DEVICE_PATH "/dev/tty-3"
```

Comment:
I just noticed this is basically a subset of the fmu module. I'm not suggesting now, but maybe we should consider breaking RC out of FMU instead of having all these different RC modules.